AWSWindowsHelpers/public/CloudFormation/New-AWSWindowsHelperCFNParameter.ps1

```diff
@@ -0,0 +1,17 @@
+function New-AWSWindowsHelperCFNParameter
```

Comment:
I think this might be overegging the pudding dude! If you splatted this function, the hashtable of params would be the same as the output! 😅